tests/layer_tests/tensorflow_tests/test_tf_FloorDiv.py

```diff
@@ -49,3 +48,75 @@ def test_add_placeholder_const_1D(self, params, ie_device, precision, ir_version
                                                           use_new_frontend=use_new_frontend),
                    ie_device, precision, ir_version, temp_dir=temp_dir,
                    use_new_frontend=use_new_frontend)
+
+
+class TestFloorDivStaticInput(CommonTFLayerTest):
+    stored_x_shape = []
+    min = -100
+    max = 200
+    step = 1
+    dtype = np.int32
+
+    def create_flordiv_tf_net(self, min, max, step, y, dtype, ir_version, use_new_frontend, x_shape=None):
+        import tensorflow as tf
+        x = np.arange(min, max, step, dtype=dtype)
+        
+        # x_shape arg can contain -1, but we need always have positive values, therefore take concrete shape value after reshape
+        x_shape = x.reshape(x_shape).shape if x_shape is not None else x.shape
+        self.stored_x_shape = x_shape
+        
+        self.min = min
+        self.max = max
+        self.step = step
+        self.dtype = dtype
+
+        tf.compat.v1.reset_default_graph()
+
+        with tf.compat.v1.Session() as sess:
+            x = tf.compat.v1.placeholder(dtype, x_shape, 'Input')
+            y = tf.constant(np.array(y).astype(dtype))
+            res = tf.raw_ops.FloorDiv(x=x, y=y)
+
+            tf.compat.v1.global_variables_initializer()
+            tf_net = sess.graph_def
+
+        ref_net = None
+
+        return tf_net, ref_net
+    
+    def _prepare_input(self, inputs_dict):
+        for input in inputs_dict.keys():
+            inputs_dict[input] = np.arange(self.min, self.max, self.step, dtype=self.dtype)
+            if self.stored_x_shape is not None:
+                inputs_dict[input] = inputs_dict[input].reshape(self.stored_x_shape)
+        return inputs_dict
+
+    test_inputs = [
+        # test for integers
+        dict(min=-20, max=20, step=1, y=[10],dtype=np.int32),
+        dict(min=-200, max=200, step=1, y=[-100], dtype=np.int32),
+        dict(min=-1000, max=1000, step=2, y=[100], dtype=np.int32),
+        dict(min=-10000, max=10000, step=100, y=[10000], dtype=np.int32),
+        dict(min=-10000, max=10000, step=100, y=[-10000], dtype=np.int32),
+        dict(min=-1e5, max=1e5, step=100, y=[1e5], dtype=np.int32),
+        dict(min=1e8, max=1e9, step=1e8, y=[1e8], dtype=np.int64),
+        dict(min=1e8, max=1e9, step=1e8, y=[-1e8], dtype=np.int64),
+        
+        # test for multidimensinal input
+        dict(min=-1000, max=1000, step=10, y=[1000], x_shape=[20, -1], dtype=np.int32),
+        dict(min=-1000, max=1000, step=10, y=[1000], x_shape=[2, 5, -1], dtype=np.int32),
+        dict(min=-1000, max=1000, step=1, y=[1000], x_shape=[2, 5, 10, -1], dtype=np.int32),
+
+        # test for floats
+        dict(min=-10, max=10, step=1, y=[10], dtype=np.float32),
+        dict(min=-1e5, max=1e5, step=100, y=[1e5], dtype=np.float64),
```

Comment:
can we remove all test cases expect the first one and add cases when y=[-5], y=[-6], y=[5], y=[6]?

Reply:
floating-point does not makes sense to verify. it is already checked.

Reply:
Now I am afraid we have execution time increase of layer tests due to big size tensors

Reply:
> Now I am afraid we have execution time increase of layer tests due to big size tensors

it might look heavy, but indeed it's not heavy at all. it's just 1000 elements, and even when i increate it by 1000 times to 1e8 execution time still is around ~4 secs!!! But anyways i removed other cases. Left only one for 1e5 just to be sure about large numbers.